test/Microsoft.OpenApi.SmokeTests/Microsoft.OpenApi.SmokeTests.csproj

```diff
@@ -1,11 +1,11 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>net462</TargetFrameworks>
+    <TargetFrameworks>net48</TargetFrameworks>
   </PropertyGroup>
 
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
-    <DefineConstants>TRACE;DEBUG;net462</DefineConstants>
+    <DefineConstants>TRACE;DEBUG;NET48</DefineConstants>
```

Comment:
i know the DefineConstants was already defined. but dont these get added by default? Want me to submit a PR (after this) that removes these?